babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    presets: [
+        ['@babel/preset-env', { targets: { node: 'current' } }],
+        '@babel/preset-typescript'],
+
+};
```

Comment:
Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
};

```

Reply:
I will fix that.